src/Mooc/Videos/Application/Modify/Description/ModifyDescriptionVideoCommandHandler.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+declare(strict_types = 1);
+
+namespace CodelyTv\Mooc\Videos\Application\Modify\Description;
+
+use CodelyTv\Mooc\Shared\Domain\Courses\CourseId;
+use CodelyTv\Mooc\Videos\Domain\VideoId;
+use CodelyTv\Mooc\Videos\Domain\VideoDescription;
+use CodelyTv\Shared\Domain\Bus\Command\CommandHandler;
+
+final class ModifyDescriptionVideoCommandHandler implements CommandHandler
+{
+    private $modifier;
+
+    public function __construct(VideoDescriptionModifier $modifier)
+    {
+        $this->modifier = $modifier;
+    }
+
+    public function __invoke(ModifyDescriptionVideoCommand $command)
+    {
+        $videoId       = new VideoId($command->videoId());
```

Comment:
Since we're on a `video` module we can directly call this variable `$id` instead of `$videoId` 😊